impl/src/main/java/com/okta/idx/sdk/impl/client/BaseIDXClient.java

```diff
@@ -112,7 +118,8 @@ public IDXClientContext interact() throws ProcessingException {
 
             Request request = new DefaultRequest(
                 HttpMethod.POST,
-                clientConfiguration.getIssuer() + "/v1/interact",
+                isRootOrgIssuer(clientConfiguration.getIssuer()) ? clientConfiguration.getIssuer() + "/oauth2/v1/interact" :
```

Comment:
Should this be `oauth2/default/v1/interact`?

Reply:
you mean for the Org issuer case?

Reply:
The possibilities for issuer URL to be set by the client are 1) Org Issuer - `https://{yourOktaDomain}` 2) Default Issuer - `https://{yourOktaDomain}/oauth2/default`. 

So for #1, the interact endpoint would be `https://{yourOktaDomain}/oauth2/v1/interact` and for #2 it would be `https://{yourOktaDomain}/oauth2/default/v1/interact`.